### DISCUSSION TITLE
It's the Twelve Days of Dancer, 2023 edition!

### DISCUSSION THREAD

**Discussion by cromedome** [PerlDancer/Dancer2#1704] [Announcements]
The 2023 Dancer Advent Calendar, [the Twelve Days of Dancer](https://advent.perldancer.org) is up and running! We hope you'll enjoy this year's version - there's a lot of fun and practical gifts to be found there.

Hot on the heels of our earlier gift (Dancer2 1.1.0) come two more plugin releases, [Dancer2::Plugin::Cache::CHI](https://metacpan.org/pod/Dancer2::Plugin::Cache::CHI) and [Dancer2::Plugin::Syntax::GetPost](https://metacpan.org/pod/Dancer2::Plugin::Syntax::GetPost).

Enjoy! Let us know what you think. Happy holidays to all! 
(and now it is time for this Dancer to enjoy a long winter's nap)

Jason/CromeDome

**Comment by ajmetz** [comment]
Anything happening for 2024 I wonder?

**Reply by cromedome** [reply]
Unfortunately I didn't have the time to organize another mini-calendar this year. Hopefully next year!
